### PR TITLE
Update core_esp8266_features.h

### PR DESCRIPTION
**Pull request by EspenT**
Added 
#include <stdint.h>
to avoid the following error:
Arduino/hardware/esp8266com/esp8266/cores/esp8266/core_esp8266_features.h: In function 'esp_get_cycle_count':
Arduino/hardware/esp8266com/esp8266/cores/esp8266/core_esp8266_features.h:59:3: error: unknown type name 'uint32_t'
   uint32_t ccount;

### CODE REVIEW

File: cores/esp8266/core_esp8266_features.h [hunk 1]
```diff
@@ -19,7 +19,7 @@
  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
  */
-
+#include <stdint.h>
```
Comment (earlephilhower) [inline cores/esp8266/core_esp8266_features.h]: Put this with the other includes below, please, with the appropriate comment.

File: cores/esp8266/core_esp8266_features.h [hunk 2]
```diff
@@ -37,27 +37,27 @@
 #include <stdlib.h> // malloc()
 #include <stddef.h> // size_t
 
-namespace arduino
+namespace arduino
```
Comment (earlephilhower) [inline cores/esp8266/core_esp8266_features.h]: Can we undo your whitespace changes here?

File: cores/esp8266/core_esp8266_features.h [hunk 3]
```diff
@@ -85,8 +85,8 @@ namespace arduino
 #define xt_rsil(level) (__extension__({uint32_t state; __asm__ __volatile__("rsil %0," __STRINGIFY(level) : "=a" (state) :: "memory"); state;}))
 #define xt_wsr_ps(state)  __asm__ __volatile__("wsr %0,ps; isync" :: "a" (state) : "memory")
 
-inline uint32_t esp_get_cycle_count() __attribute__((always_inline));
-inline uint32_t esp_get_cycle_count() {
+inline uint32_t esp_get_cycle_count() __attribute__((always_inline));
```
Comment (earlephilhower) [inline cores/esp8266/core_esp8266_features.h]: And here...

**Comment by devyte** [comment]
@EspenT ping

**Comment by EspenT** [comment]
Sorry for the delay. I'm learning to use git. Is it OK now ?

**Comment by devyte** [comment]
@EspenT not ok, something is messed up, now there are conflicts. Could you please make a new PR?

**Comment by earlephilhower** [comment]
Closing as fixed by #6793

**Pull request closed without merging**